src/features/projects/projectsSlice.ts

```diff
@@ -211,6 +212,11 @@ export const projectsSlice = createSlice({
               target: 'authWebhookMethods',
               message: description,
             };
+          } else if (field === 'clientDeactivateThreshold') {
```

Comment:
The error field begins with a capital letter.
`clientDeactivateThreshold` -> `ClientDeactivateThreshold` 

<img width="663" alt="image" src="https://user-images.githubusercontent.com/81357083/221125817-31850775-0584-4bde-bef6-d0e09e4ccb1b.png">

Reply:
Opps, small typos. Thank you for letting me know.